bin/backdrop/backdrop.js

```diff
@@ -0,0 +1,109 @@
+// Usage: npm run backdrop --input path/to/icon.png --output path/to/output.png
+// Warning: canvas isn't compiled for macOS-arm64, follow compiling instructions:
+//  https://www.npmjs.com/package/canvas#compiling
+const { existsSync, mkdirSync, createWriteStream } = require('fs');
+const trianglify = require('trianglify');
+const { program } = require('commander');
+const { loadImage } = require('canvas');
+const { join, dirname } = require('path');
+
+program
+  .version('0.0.0')
+  .description('A small tool to generate geometric backdrops layered with an icon.')
+  .option('-i, --input <filename>', 'Specify the icon/input file', 'static/img/conda_logo_c.png')
+  .requiredOption('-o, --output <filename>', 'Specify the output file')
+  .parse();
+const options = program.opts();
+
+const input = join("..", "..", options.input);
+const output = join("..", "..", options.output);
+
+// Randomize color function
+const colorFunctions = Object.values(trianglify.colorFunctions);
+const colorFunction = colorFunctions[Math.floor(Math.random() * colorFunctions.length)];
+
+// Generate a canvas using trianglify
+const canvas = trianglify({
+  width: 1200,
+  height: 630,
```

Comment:
What is the "correct" size for a banner?

Reply:
The exact size doesn't matter, but it should be close to a 2:1 aspect ratio